test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -377,3 +377,92 @@ TEST_F(CLASSNAME(TestSubscriptionUse, RMW_IMPLEMENTATION), count_mismatched_subs
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
   EXPECT_EQ(0u, publisher_count);
 }
+
+TEST_F(CLASSNAME(TestSubscriptionUse, RMW_IMPLEMENTATION), take_sequence) {
+  rmw_ret_t ret;
+  size_t count = 1u;
+  size_t taken;
+  rcutils_allocator_t allocator = rcutils_get_default_allocator();
+  rmw_message_sequence_t sequence = rmw_get_zero_initialized_message_sequence();
+  ret = rmw_message_sequence_init(&sequence, count, &allocator);
```

Comment:
Like this, may need extra new line:
```suggestion
rmw_ret_t ret = rmw_message_sequence_init(&sequence, count, &allocator);
```

Reply:
Updated on dc8ab77